appveyor.yml

```diff
@@ -6,12 +6,22 @@ before_build:
   - nuget restore nunit.sln
 
 build_script: 
-  - msbuild nunit.sln /p:Configuration=Release /t:Rebuild
+  - ps: |
+      .\appveyor.ps1
+      if ($lastexitcode -ne 0){ exit $lastexitcode }
   
-test_script:
-  - msbuild NUnit.proj /t:TestAll /p:Configuration=Release /p:ResultFormat=nunit2 /p:ContinueOnFailure=ErrorAndContinue
+#test_script:
+#  - msbuild NUnit.proj /t:TestAll /p:Configuration=Release /p:ResultFormat=nunit2 /p:ContinueOnFailure=ErrorAndContinue
 #  - ps: $wc = New-Object 'System.Net.WebClient'
 #  - ps: $dir = "bin\Release\Results"
 #  - ps: $uri = "https://ci.appveyor.com/api/testresults/nunit/$($env:APPVEYOR_JOB_ID)"
 #  - ps: foreach($item in (dir $dir "*.xml")) { $wc.UploadFile($uri, $item.FullName) }
   
+artifacts:
```

Comment:
Does on branch work with artifacts, or just deployments? If not, we could use a conditional build configuration, http://www.appveyor.com/docs/branches#conditional-build-configuration

```YAML
on:
      branch: master
```